### ISSUE TITLE
"Fill match arms" doesn't support match ergonomics

### ISSUE THREAD

**Issue by Riateche** [intellij-rust/intellij-rust#3327]
The plugin doesn't suggest to "fill match arms" in this code:
```rust
enum A {
    B, C,
}
fn x(a: &A) {
    match a {
        [cursor here]    
    }
}
```
However, the suggestion appears if I change the code to `match *a`. Now both variants compile (if match arms are filled), so the suggestion should work in both cases.

**Comment by dima74** [comment]
@Riateche looks like it is fixed now (version 0.2.124.3148-202-nightly):

![Peek 2020-06-09 18-31](https://user-images.githubusercontent.com/6505554/84167866-30440c80-aa90-11ea-9e35-a69504262c03.gif)

**Comment by Riateche** [comment]
Yes, it works now.
